### ISSUE TITLE
[Suggestion] Multiserver exploit

### ISSUE THREAD

**Issue by BookWyrm114** [MeteorDevelopment/meteor-client#2469]
### Describe the feature

A feature allowing you to abuse this wonderful exploit: https://youtu.be/EEG_c3ycDg4 and join 2 servers at once. Possibly 3, 4, 5, or an infinite amount of servers could be joined at once by intentionally causing this glitch, only 2 simultaneous servers are reproducible in vanilla, but it seems like more should be possible if the exploit can be reproduced intentionally.

This could work as a button on the multiplayer menu that, when pressed, allows you to select multiple servers, meteor will then abuse the exploit to join them all.

### Before submitting a suggestion

- [X] This feature doesn't already exist in the client.
(I have checked every module and their settings on the **latest dev build**)

- [X] This wasn't already suggested.
(I have checked every suggestion on github)

- [X] This feature is a valid suggestion.
(I have read the FAQs and Guides)

**Comment by RickyTheRacc** [comment]
This only works if you have creative on the server you're trying to affect, so no

**Comment by BookWyrm114** [comment]
No, it still works, you just can’t transfer items over. It’s still a fun module, and also allows for some basic x-raying on servers with anti-Xray, as well as a few other useful/fun bugs

**Comment by 19MisterX98** [comment]
Your anti xray bypass is called wallhack which is included in meteor
